airflow/docker.go

```diff
@@ -316,6 +323,26 @@ func checkServiceState(serviceState, expectedState string) bool {
 	return scrubbedState == expectedState
 }
 
+func (d *DockerCompose) webserverHealthCheck() error {
+	err := d.composeService.Events(context.Background(), d.projectName, api.EventsOptions{
+		Services: []string{config.CFG.WebserverContainerName.GetString()}, Consumer: func(event api.Event) error {
+			if event.Status == "health_status: healthy" {
```

Comment:
can this status be a const string? followed by the exec_die as well?

Reply:
make sense, will move it to a package level constant

Reply:
moved to a package level constant